sdk/core/azure-core/inc/azure/core/operation.hpp

```diff
@@ -53,10 +53,10 @@ namespace Azure { namespace Core {
 
     /**
      * @brief Get the raw HTTP response.
-     * @return A pointer to #Azure::Core::Http::RawResponse.
+     * @return A #Azure::Core::Http::RawResponse.
```

Comment:
```suggestion
     * @return A reference to an #Azure::Core::Http::RawResponse.
```
OR, if you really want to be brief.
```suggestion
     * @return An #Azure::Core::Http::RawResponse.
```